java/developer-guide/working-with-microsoft-graph-client/microsoft-graph-utility-features/_index.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Microsoft Graph Utility Features
+type: docs
+weight: 10
+url: /java/microsoft-graph-utility-features/
+---
+
+
+## **Creating Project in Azure Active Directory Admin Center**
+
+A Project is to be created on Azure Active Directory admin center for a user having MS Office account.
+### **Steps to Create a Project in Azure Active Directory Admin Center**
+
+Following is a step by step tutorial for creating a project in Azure Active Directory admin center.
+
+#### 1. Go to link <https://aad.portal.azure.com/> and login using your MS Office credentials.
```

Comment:
Remove the link from the header. It would be better if you place that link right after the header.